src/server/utils/cacheCDNModules.js

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2023 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import path from 'path';
+import fs from 'fs';
+import chalk from 'chalk';
+
+export const getUserHomeDirectory = () => process.env.HOME || process.env.USERPROFILE;
+export const cacheFileName = '.one-app-module-cache';
+export const moduleCacheFileName = '.one-app-module-map-cache';
+export const oneAppDirectoryName = '.one-app';
+export const oneAppDirectoryPath = path.join(getUserHomeDirectory(), oneAppDirectoryName);
+export const oneAppModuleCachePath = path.join(oneAppDirectoryPath, cacheFileName);
+
+// show cache size and how to delete info on start
+export const showCacheInfo = () => {
+  fs.stat(oneAppModuleCachePath, (error, stats) => {
```

Comment:
```suggestion
  fs.stat(oneAppModuleCachePath, (error, stats) => {
```
Can we switch this to `async-await`